server/routes/clientRouter.js

```diff
@@ -2,7 +2,7 @@ import express from "express"
 
 const router = new express.Router()
 
-const clientRoutes = ["/", "/pets", "/pets/:type", "/pets/:type/:id"]
+const clientRoutes = ["/", "/pets", "/pets/:type", "/pets/:type/:id", "/adoptions/new"]
```

Comment:
again related to story #4 and will want to remove from this PR

Reply:
Story 4 is in main